for-research/phoenix/get_json_phoenix.py

```diff
@@ -220,10 +307,10 @@ def json_format(files: list[str],
                     eventList += [ ( int(cut_line.split(',')[0].strip()),int(cut_line.split(',')[1].strip()) ) ]
                 else:
                     print(f'Could not parse {cut_line} in {eventListFile} as run, event pair')
-        if ntuple:
-            print('FYI: When running on ntuples, run numbers are not available, and event numbers correspond to TTree entry numbers')
-            # Modify the event list accordingly
-            eventList = [ (1,x[1]) for x in eventList ]
+            if ntuple:
```

Comment:
This indentation doesn't make a difference, I think — right? Something I'm missing? This just seems to put it in the file reading scope, which shouldn't hurt anything but I hope doesn't fix a bug that I'm not seeing?